pkg/operator/hostetcdendpointcontroller/members.go

```diff
@@ -163,3 +177,12 @@ func (h *healthyEtcdMemberGetter) Endpoints() ([]string, error) {
 
 	return endpoints, nil
 }
+
+//TODO add to util
+func IsIP(addr string) bool {
+	ip := net.ParseIP(addr)
```

Comment:
Can we please use the variable in the if block below?

Reply:
done